go/pkg/pwagent/daemon.go

```diff
@@ -57,27 +64,116 @@ func Daemon(ctx context.Context, cli *client.Client, logger *zap.Logger) error {
 	fmt.Println("api instances", godev.PrettyJSON(apiInstances))
 
 	// fetch local info from docker daemon
-	dockerInstances, err := pwcompose.GetPathwarInfo(ctx, cli)
+	pathwarInfo, err := pwcompose.GetPathwarInfo(ctx, cli)
 	if err != nil {
 		return errcode.ErrComposeGetPathwarInfo.Wrap(err)
 	}
-	fmt.Println("info", godev.PrettyJSON(dockerInstances))
+	fmt.Println("info", godev.PrettyJSON(pathwarInfo))
 
+	agentOpts := AgentOpts{
+		DomainSuffix:      "local",
+		HostIP:            "0.0.0.0",
+		HostPort:          "8000",
+		ModeratorPassword: "",
+		Salt:              "",
```

Comment:
salt is too ambiguous prefix it with its goal

Reply:
PrefixSalt ?